src/componentes/unmounted/lista-nominal/print/aux/StatusAtendimentoOdontologico.jsx

```
@@ -0,0 +1,31 @@
+})=> {
+  const descricao = useMemo(() => {
+    return identificacao_atendimento_odontologico.find(item => item.id_atendimento_odontologico == value).atendimento_odontologico_descricao;
+  }, [identificacao_atendimento_odontologico, value]);
+
+  return (
+    <div style={{
+      ...STYLE[value],
+      width: "55%",
+      fontSize: orientacao === "retrato" && "9px"
+    }}>
+      {value == 1 && <img src={icones.check_simbolo} />}
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)